code/modules/mob/living/basic/minebots/minebot_ai.dm

```diff
@@ -0,0 +1,226 @@
+/datum/ai_controller/basic_controller/minebot
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic,
+		BB_PET_TARGETTING_DATUM = new /datum/targetting_datum/not_friends,
+		BB_BLACKLIST_MINERAL_TURFS = list(/turf/closed/mineral/gibtonite),
+		BB_AUTOMATED_MINING = FALSE,
+	)
+
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/pet_planning,
+		/datum/ai_planning_subtree/basic_ranged_attack_subtree/minebot,
+		/datum/ai_planning_subtree/find_and_hunt_target/consume_ores/minebot,
+		/datum/ai_planning_subtree/minebot_mining,
+		/datum/ai_planning_subtree/locate_dead_humans,
+	)
+
+///find dead humans and report their location on the radio
+/datum/ai_planning_subtree/locate_dead_humans/SelectBehaviors(datum/ai_controller/controller, seconds_per_tick)
+	var/mob/living/dead_target = controller.blackboard[BB_NEARBY_DEAD_MINER]
+
+	if(QDELETED(dead_target))
+		controller.queue_behavior(/datum/ai_behavior/find_and_set/unconscious_human, BB_NEARBY_DEAD_MINER, /mob/living/carbon/human)
+		return
+
+	controller.queue_behavior(/datum/ai_behavior/send_sos_message, BB_NEARBY_DEAD_MINER)
+	return SUBTREE_RETURN_FINISH_PLANNING
+
+/datum/ai_behavior/find_and_set/unconscious_human/search_tactic(datum/ai_controller/controller, locate_path, search_range)
+	for(var/mob/living/carbon/human/target in oview(search_range, controller.pawn))
+		if(target.stat >= UNCONSCIOUS && target.mind)
+			return target
+
+/datum/ai_behavior/send_sos_message
+	behavior_flags = AI_BEHAVIOR_CAN_PLAN_DURING_EXECUTION
+	action_cooldown = 5 MINUTES
```

Comment:
five minutes seems like a bit much for someone who might be in critical condition, no? i get not wanting to spam supply comms but i don't think this makes sense from an in-universe perspective. i think slashing it down to `2 MINUTES` would be okay unless you have further thoughts on the matter